code/controllers/subsystems/initialization/codex.dm

```diff
@@ -113,9 +113,9 @@ SUBSYSTEM_DEF(codex)
 			for(var/entry_title in entries_by_string)
 				var/datum/codex_entry/entry = entries_by_string[entry_title]
 				if(findtext(entry.display_name, searching) || \
-				 findtext(entry.lore_text, searching) || \
-				 findtext(entry.mechanics_text, searching) || \
-				 findtext(entry.antag_text, searching))
+					findtext(entry.lore_text, searching) || \
```

Comment:
this doesnt seem correct, these are all part of the same IF and shouldn't be indented. They're not inside the conditional area